config/prometheus/kustomization.yaml

```diff
@@ -5,3 +5,12 @@ namespace: kueue-system
 namePrefix: kueue-
 resources:
 - ../components/prometheus
+
+commonLabels:
```

Comment:
why add this here?

Reply:
Prometheus files were modified here https://github.com/kubernetes-sigs/kueue/pull/1695/files#diff-e7bb691a4b46f74d3503c16fd1e5e62e7f292bd15e9be5f1667f0c5e376c84ce

Don't we need to restore the labels in there as well?

Reply:
Yes, prometheus files were modified, but not through kustomize, because kustomize labels don't apply to ServiceMonitor.

However, because you are adding both old labels and new labels, I think the ServiceMonitor might just work with the current selectors in the `main` branch.

Could you give it a manual test?
Run https://kueue.sigs.k8s.io/docs/installation/#add-metrics-scraping-for-prometheus-operator
 and then verify in the prometheus UI that the metrics show up.

Reply:
I've added Prometheus on the fix branch and I can see metrics:

![screencapture-localhost-9090-graph-2024-05-27-22_42_13](https://github.com/kubernetes-sigs/kueue/assets/22640625/e8c14e7b-d582-4b51-80fe-7e9f06a2c81c)

Reply:
That's not a kueue metric. I think you should be able to go to the Status pane and see which metrics kueue is exporting.

And if it's not exporting anything, try creating one job, so that Kueue at least records one metric for that admitted workload.

Reply:
After creating some jobs here are a couple of metrics:

`kueue_admitted_workloads_total`:

![screencapture-localhost-9090-graph-2024-05-28-11_24_08](https://github.com/kubernetes-sigs/kueue/assets/22640625/64cc4ba5-f2cd-42c5-8ce0-a8a41e274ee3)

`kueue_pending_workloads`:

![screencapture-localhost-9090-graph-2024-05-28-11_23_56](https://github.com/kubernetes-sigs/kueue/assets/22640625/8f4de57e-1e2f-4956-b19a-1565fab97df8)

Reply:
Thanks for checking!